en/competition/index.md

```diff
@@ -7,34 +7,34 @@ ref: competition
 
-## Robocup@Home in brief
+## Robocup@Home in a few lines
 
-The RoboCup@Home competition is held during the Robocup, the most important competition of Robotics in the world. The latter also presents the Soccer, Rescue and Junior categories. This event held once a year, brings together university and college from across the world.
+The RoboCup@Home is held during the Robocup, the most important robotics competition in the world. The latter also showcase the Soccer, Rescue, and Junior categories. This event, held once a year, brings together university and college from across the world.
 
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/YpjeNa8BAYg" frameborder="0" allowfullscreen></iframe></center><br>
-RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in the tasks of everyday life. To determine the best robots, the teams move to their robot a series of tests to measure their skills and performance in an environment simulating a House. The main axes of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, object manipulation and artificial intelligence.
+RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in everyday tasks. To find the best robots, the teams make their robot do a series of tests to measure their skills and performance in an house like environment. The main fields of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, objects manipulation and artificial intelligence.
 
-## Déroulement de la compétition
+## During the competition
 
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/-oGeU-58wrQ" frameborder="0" allowfullscreen></iframe></center><br>
-The events are presented in the form of scenario spread out over two levels. Thus, After passing the qualification phase, the robots are confronted with scenarios of the level 1. The robots having the most points access level 2. The five teams who have accumulated the most points during the scenarios past access to the final round. The winning robot is the one who has accumulated the most points throughout his career.
+The events are presented in the form of scenarios spread out over two levels of difficulty. Thus, after passing the qualification phase, the robots are confronted with level 1 scenarios. The robots that scored the most access level 2. The five teams who have accumulated the most points during the past scenarios access to the final round. The winning robot is the one who has accumulated the most points throughout the different heats.
```

Comment:
The five teams who have accumulated the most points during the competition go on to the final round. The winning robot is the one who has accumulated the most points throughout the different stages.